### PR TITLE
Raiyan#269 todo app level-0

### PR DESCRIPTION
**Pull request by raiyan-hosn**
Fixes #

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## What was done
Implemented todo app with CSS and usestate. 
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: src/components/TaskInput/TaskInput.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      setTask("");
+      taskRef.current.blur();
+    } else if (e.key === "Escape") {
+      setTask("");
+      taskRef.current.blur();
+    }
+  };
+  return (
+    <div className="task-input">
+      <form className="task-input" onSubmit={(e) => handleTaskSubmit(e)}>
```
[the remark this answers is not in the document]
Comment (raiyan-hosn) [reply src/components/TaskInput/TaskInput.js]: Modified.

File: src/components/TaskList/TaskItem/TaskItem.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="20"
+              height="20"
+              fill="currentColor"
+              className="bi bi-check"
+              viewBox="0 0 16 16"
+            >
+              <path d="M10.97 4.97a.75.75 0 0 1 1.07 1.05l-3.99 4.99a.75.75 0 0 1-1.08.02L4.324 8.384a.75.75 0 1 1 1.06-1.06l2.094 2.093 3.473-4.425a.267.267 0 0 1 .02-.022z" />
+            </svg>
```
[the remark this answers is not in the document]
Comment (raiyan-hosn) [reply src/components/TaskList/TaskItem/TaskItem.js]: shifted them to the icons folder.

File: src/components/TaskList/TaskItem/TaskItem.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import "./TaskItem.css";
+function TaskItem({ handleDelete, task, index }) {
+  const [isComplete, setIsComplete] = useState(false);
+  const handleComplete = () => {
+    setIsComplete(true);
+  };
```
[the remark this answers is not in the document]
Comment (raiyan-hosn) [reply src/components/TaskList/TaskItem/TaskItem.js]: Modified.

**Review by tusher-A — APPROVED** [review]
LGTM

**Pull request merged**